ui/imports/utils/Constants.qml

```diff
@@ -613,16 +613,37 @@ QtObject {
     readonly property int communityChatInvitationOnlyAccess: 2
     readonly property int communityChatOnRequestAccess: 3
 
-    readonly property int activityCenterNotificationTypeOneToOne: 1
-    readonly property int activityCenterNotificationTypeGroupRequest: 2
-    readonly property int activityCenterNotificationTypeMention: 3
-    readonly property int activityCenterNotificationTypeReply: 4
-    readonly property int activityCenterNotificationTypeContactRequest: 5
-    readonly property int activityCenterNotificationTypeCommunityInvitation: 6
-    readonly property int activityCenterNotificationTypeCommunityRequest: 7
-    readonly property int activityCenterNotificationTypeCommunityMembershipRequest: 8
-    readonly property int activityCenterNotificationTypeCommunityKicked: 9
-    readonly property int activityCenterNotificationTypeContactVerification: 10
+    enum ActivityCenterGroup {
+        All = 0,
+        Mentions = 1,
+        Replies = 2,
+        Membership = 3,
+        Admin = 4,
+        ContactRequests = 5,
+        IdentityVerification = 6,
+        Transactions = 7,
+        System = 8
+    }
+
+    enum ActivityCenterNotificationType {
+        NoType = 0,
+        NewOneToOne = 1,
+        NewPrivateGroupChat = 2,
+        Mention = 3,
+        Reply = 4,
+        ContactRequest = 5,
+        CommunityInvitation = 6,
+        CommunityRequest = 7,
+        CommunityMembershipRequest = 8,
+        CommunityKicked = 9,
+        ContactVerification = 10
+    }
+
+    enum ActivityCenterReadType {
+        Read = 1,
+        Unread = 2,
+        All = 3
+    }
```

Comment:
Hmm, wouldn't ActivityCenterStore be a better place for these enums? :thinking: Afaik they are not used application-wide (outside of the AC stuff), wdyt?

Reply:
Yes, from my point of view ActivityCenterStore is the best place for that enums, here i just followed conventions. I'll move them to  ActivityCenterStore

Reply:
fixed

Reply:
Really? :)

Reply:
just pushed :)